modules/ingester/ingester.go

```diff
@@ -183,49 +190,38 @@ func (i *Ingester) starting(ctx context.Context) error {
 		}
 	}
 
+	// accept traces
 	i.pushErr.Store(nil)
 
+	// start flushing traces to wal
+	close(i.cutToWalStart)
+
 	return nil
 }
 
 func (i *Ingester) loop(ctx context.Context) error {
```

Comment:
```suggestion
func (i *Ingester) running(ctx context.Context) error {
```

`loop` never truly looped, but now it even makes less sense than before

Reply:
good call. i think that also makes it more consistent with the other modules